### PR TITLE
Monitor multiple areas with Brottsplatskartan component

### PR DESCRIPTION
**Pull request by chrillux**
It's now possible to monitor multiple areas with configuration.

**Description:**

Rebuilt Brottsplatskartan and put the polling logic in the component. The sensor is now listening on events from the component.

New configuration structure: Using the Brottsplatskartan component that has a sensor platform. The "areas" parameter is new and enables you to monitor multiple areas.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#23546

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/brottsplatskartan.markdown [hunk 1]
```diff
@@ -2,15 +2,15 @@
 layout: page
 title: "Brottsplatskartan"
 description: "Instructions on how to integrate brottsplatskartan.se into Home Assistant."
-date: 2018-12-02 20:00
+date: 2019-05-11 22:30
```
Comment (klaasnicolaas) [inline source/_components/brottsplatskartan.markdown]: Please do not change this

File: source/_components/brottsplatskartan.markdown [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
 sidebar: true
 comments: false
 sharing: true
 footer: true
 ha_category:
   - Social
 logo: brottsplatskartan.png
-ha_release: 0.85
+ha_release: 0.94
```
Comment (klaasnicolaas) [inline source/_components/brottsplatskartan.markdown]: Same

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/brottsplatskartan.markdown (hunk 1 above)
Comment (chrillux) [reply source/_components/brottsplatskartan.markdown]: Thanks, changed it back to the old value.

File: source/_components/brottsplatskartan.markdown (hunk 2 above)
Comment (chrillux) [reply source/_components/brottsplatskartan.markdown]: Thanks, changed it back to the old value.

**Comment by frenck** [comment]
Closing this PR, since the parent PR has been closed.

**Pull request closed without merging**